internal/config/default_config.go

```diff
@@ -134,5 +40,66 @@ var (
 		},
 	}
 
-	DefaultEvents = []string{"ADD", "UPDATE", "DELETE"}
+	DefaultEvents     = []string{"ADD", "UPDATE", "DELETE"}
+	ExemptedResources = []string{"selfsubjectrulesreviews", "localsubjectaccessreviews", "bindings", "selfsubjectaccessreviews", "subjectaccessreviews", "tokenreviews", "componentstatuses", "flowschemas", "prioritylevelconfigurations"}
 )
+
+func PopulateDefaultResources(discoveryClient discovery.DiscoveryInterface) error {
+	// Get all resources in the cluster
+	clusterResources, namespacedResources, err := getAllResources(discoveryClient)
+	if err != nil {
+		fmt.Printf("Error getting all resources: %v\n", err)
+		return ErrInitConfig(err)
+	}
+	var localResources []PipelineConfig
+
+	for _, v := range namespacedResources {
+		localResources = append(localResources, PipelineConfig{Name: v, Events: DefaultEvents, PublishTo: "meshery.meshsync.core"})
+	}
+	Pipelines[LocalResourceKey] = localResources
+
+	var globalResources []PipelineConfig
+
+	for _, v := range clusterResources {
+		globalResources = append(globalResources, PipelineConfig{Name: v, Events: DefaultEvents, PublishTo: "meshery.meshsync.core"})
+	}
+	Pipelines[GlobalResourceKey] = globalResources
+
+	return nil
+}
+func getAllResources(discoveryClient discovery.DiscoveryInterface) ([]string, []string, error) {
+
+	_, groupList, err := discoveryClient.ServerGroupsAndResources()
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var clusterResources []string
+	var namespacedResources []string
+
+	for _, group := range groupList {
+		for _, resource := range group.APIResources {
+			if strings.Contains(resource.Name, "/") {
```

Comment:
What is this for?
Does this mean resources under `apps/v1` , `networking.k8s.io/v1` ... not to be watched for changes inside the cluster?